### ISSUE TITLE
Ability to show URL associated with task

### ISSUE THREAD

**Issue by jelmer** [pimutils/todoman#313]
It would be great if there was a way to view the URL associated with a task in todoman.

Better yet would be if there was some command to open the URL in a browser.

**Comment by WhyNotHugo** [comment]
Yeah, this doesn't sound like a big deal; `todo show` should include the url.

We don't actually show `url` anywhere, so we'd also have to include this in edit/etc.
